src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetCultureCommand.cs

```diff
@@ -1,23 +1,94 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.Globalization;
 using System.Management.Automation;
 
 namespace Microsoft.PowerShell.Commands
 {
     /// <summary>
-    /// Returns the thread's current culture.
+    /// Returns:
+    ///     - the thread's current culture
+    ///     - culture by name
+    ///     - list of all supported cultures.
     /// </summary>
-    [Cmdlet(VerbsCommon.Get, "Culture", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113312")]
+    [Cmdlet(VerbsCommon.Get, "Culture", DefaultParameterSetName = DefaultParameterSet, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113312")]
     [OutputType(typeof(System.Globalization.CultureInfo))]
     public sealed class GetCultureCommand : PSCmdlet
     {
+        private const string DefaultParameterSet = "Default";
+        private const string NameParameterSet = "Name";
+        private const string ListParameterSet = "List";
+
+        /// <summary>
+        /// Gets or sets culture names for which CultureInfo values are returned.
+        /// </summary>
+        [Parameter(ParameterSetName = NameParameterSet, Position = 0, ValueFromPipeline = true)]
+        [ValidateNotNull]
+        public string[] Name { get; set; }
+
         /// <summary>
-        /// Output the current Culture info object.
+        /// Gets or sets a switch to return predefined cultures.
+        /// By default we return cultures in current state (with custom changes).
+        /// With the switch on, we return predefined, original cultures.
         /// </summary>
-        protected override void BeginProcessing()
+        [Parameter(ParameterSetName = NameParameterSet)]
+        [ValidateNotNull]
+        public SwitchParameter Predefined { get; set; }
```

Comment:
As discussed, I don't think we need this switch: user overrides only ever exist for the _current_ culture, so `Get-Culture` without arguments returns the current culture _with_ overrides. 

It's fine to let `Get-Culture` with `-Name` return the  culture _without overrides_, if the argument happens to be the current culture.

That way, user can gain access to either, as desired - we just need to document it as such, along the lines of:

"Use `Get-Culture` to get the current culture _with_ overrides; use `Get-Culture $PSCulture` if you want the _original_ definition."